### PR TITLE
Add Voodoo setup case study

### PR DESCRIPTION
**Pull request by zacsimile**

### CODE REVIEW

File: docs/source/case_studies/setup_voodoo.rst [hunk 1]
```diff
@@ -0,0 +1,125 @@
+Case Study: Setting up the an Axially Swept Light-Sheet Microscope
+==================================================================
+
+The key to properly setting up the navigate software is to sequentially enable select
+devices, and troubleshoot each one independently. By carefully and methodically
+adding devices, and checking that they are functional, one can be confident that the
+entire system is working as intended. In this example, we will be using a Hamamatsu
+Flash 4.0 camera. However, you may also use another Hamamatsu SCMOS model, or a
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst:8]: sCMOS

File: docs/source/case_studies/setup_voodoo.rst [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+evaluate the card's functionality with an oscilloscope.
+
+#. In the `hardware` and `microscope` sections of the `configuration.yaml` file, change
+the `type` to `NI`. You will also need to hard-wire the master_trigger_out_line to the
+trigger_source. The identity of these pins can be found in the NI MAX software by right-
+clicking on the device and selecting `Device Pinouts`. You will also need to make
+sure that the identity of the pinouts is correct in the `configuration.yaml` file.
+Most commonly, NI cards default to a device name such as `Dev1`. You can change this
+name in NI MAX, or leave it as is, but whatever you do it has to match the name in
+the `configuration.yaml` file (e.g., PXI6259/port0/line1 if the name of the device is
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst]: `PXI6259/port0/line1`

File: docs/source/case_studies/setup_voodoo.rst [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+PXI6259 and the pinout is port0/line1).
+
+#. Open the `navigate` software in the standard operating mode, select the `continuous`
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst]: "Continuous Scan"

File: docs/source/case_studies/setup_voodoo.rst [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+mode, and press `acquire`. If the software is operating as expected, it should display
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst]: :guilabel:`Acquire`

File: docs/source/case_studies/setup_voodoo.rst [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+file and make sure that the master_trigger_out_line is connected to the trigger_source.
+
+#. Next, we will add the camera. The camera must be connected to the computer via a USB
+or the dedicated frame-grabber cable. The camera drivers must also be installed. The
+camera drivers can be found on the `DCAM-API Website <https://dcam-api.com>`_.
+
+#. Update the camera type to `HamamatsuOrca` and input the correct serial number,
+which can be found on the camera label or through the DCAMConfigurator or HCImage.
+You will also need to connect the `camera_trigger_out_line` on the data acquisition
+card to the `Ext. Trigger` on the camera.
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst]: BNC port labeled "Ext. Trig." on the back of the camera

File: docs/source/case_studies/setup_voodoo.rst [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+
+#. Restart the software and begin an acquisition in the `continuous` mode. The camera
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst]: "Continuous Scan"

File: docs/source/case_studies/setup_voodoo.rst [hunk 7]
```diff
@@ -0,0 +1,125 @@
...
+should not be delivering frames to the software.
+
+#. Set up the Filter Wheel. First, identify the comport via the device manager
+on Windows. Change the `filter_wheel` type to `SutterFilterWheel` or equivalent in
+the `hardware` and `microscope` sections of the configuration file, and
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst]: `microscopes`

File: docs/source/case_studies/setup_voodoo.rst [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+provide the necessary information for that filter wheel device (e.g., baudrate). At
+this point, you can also name each filter if desired under the `available_filters`
+section.
+
+#. Restart the software and select multiple channels in the channel settings tab,
```
Comment (zacsimile) [inline docs/source/case_studies/setup_voodoo.rst:81]: :guilabel:`Channel Settings`

**Review by AdvancedImagingUTSW — APPROVED** [review]

**Pull request merged**